### ISSUE TITLE
Anyway to get the days a player did not play

### ISSUE THREAD

**Issue by samwsweet** [swar/nba_api#322]
Right now pulling a players game logs just returns that logs of the games where a player logged at least 1 minute. Is there anyway to get the games they did not play and possibly the reason why? Is there also a way to get what games a player started?

**Comment by swar** [comment]
> Anyway to get the days a player did not play

Yes you can get all the games for the season for that player and then use that information to get the days they did not play.

> Is there anyway to get the games they did not play

Yes, you can get all games for a team and then cross reference it with the games a player did play.

> possibly the reason why?

Don't think this is possible.

> Is there also a way to get what games a player started?

Not sure, but I believe this is possible.
